service/provider/src/main/java/gov/va/vro/service/provider/camel/MasIntegrationProcessors.java

```diff
@@ -150,4 +151,23 @@ public static Processor slackEventProcessor(String routeId, String message) {
       exchange.getIn().setBody(AuditEvent.fromAuditable(auditable, routeId, msg));
     };
   }
+
+  /**
+   * This sets up a skeleton lighthouse HealthDataAsessment object in the event of a timeout from
+   * lighthouse. We know the fields we MUST have for evidence merge from the properties we saved on
+   * the exchange.
+   *
+   * @return
+   */
+  public static Processor lighthouseContinueProcessor() {
```

Comment:
For reference https://amida.atlassian.net/browse/MCP-2547 is what's being addressed here.
It also implies 
https://amida.atlassian.net/browse/MCP-2548 to deal with the fact we're doing nearly the same thing in two code locations.